iscript/src/iscript/constants.py

```diff
@@ -22,6 +22,9 @@
         "zipfile_cmd": "ditto",
         "create_pkg": False,
         "hardened_runtime_only_files": ["wg", "wireguard-go"],
+        "profiles": {
+            "path": "local_profile_path"
+        }
```

Comment:
Further up in this file I see we have `designated_requirements` entries for each product. I presume that under the new signing formats these are just going to be completely ignored, since they're coming in (directly or indirectly) via the payload instead? If that's the case, we ought to make that clear. A few suggestions (let's do at least one...but I'll leave it up to you how far to go):
* Rename to `non_hardened_designated_requirements`
* Note in the log with each hardened signing that the `designated_requirements` from the product config is being ignored
* Add new product configs that don't have this at all, eg: `mozillavpn-hardened`.

Reply:
I believe for now `designated_requirements` will still be hard-coded in constants due to its sensitivity.
The "profiles" object here was actually left in by mistake.. Oops :)